app/src/main/res/layout/dialog_switch_preference.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="24dp">
+
+    <Switch
+        android:id="@+id/switch1"
```

Comment:
Why the `1`?

Reply:
The compiler complains that switch is reserved and cannot be used and that is the name it suggested.

Reply:
Then I would suggest something like `dialogSwitch`. The number makes one wonder if there are multiple switches.